third_party/closure-compiler/src/AmpCommandLineRunner.java

```diff
@@ -0,0 +1,36 @@
+import java.util.HashSet;
+import java.util.List;
+
+import com.google.javascript.jscomp.CommandLineRunner;
+import com.google.javascript.jscomp.CompilerOptions;
+import com.google.common.collect.ImmutableList;
+
+public class AmpCommandLineRunner extends CommandLineRunner {
+
+  private static final List<String> LOG_STRIP_TYPES = ImmutableList.of(
+      "dev.fine",
+      "dev.assert");
+
+  protected AmpCommandLineRunner(String[] args) {
+    super(args);
+  }
+
+  @Override
+  protected CompilerOptions createOptions() {
+    CompilerOptions options = super.createOptions();
```

Comment:
How do we test this? Should we first try it w/o LOG_STRIP_TYPES and see if the result is equivalent?